x-pack/plugins/reporting/public/components/report_listing.tsx

```diff
@@ -21,10 +21,12 @@ import {
   EuiTitle,
   EuiToolTip,
 } from '@elastic/eui';
-import { Poller } from '../../../../common/poller';
-import { JobStatuses } from '../constants/job_statuses';
-import { downloadReport } from '../lib/download_report';
-import { jobQueueClient, JobQueueEntry } from '../lib/job_queue_client';
+
+import { ToastsSetup, ApplicationStart } from '../../';
```

Comment:
Is it still the case that we can't import these from `src/core/public`? IIRC that limitation was going on because this code was in `legacy` earlier

Reply:
Yeah, this was to get around not being able to import in sub-modules. I'm not sure that restriction is lifted now, but I'll check.

Reply:
I believe this can switch to `import { ToastsSetup, ApplicationStart }  from 'src/core/public';`